ferrocene/doc/requirements/src/self-test.rst

```diff
@@ -0,0 +1,209 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+`ferrocene-self-test` tool
+==========================
+
+General checks
+--------------
+
+Checks for executables
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. note::
+
+   **executable** refers to the following:
+
+   - ``rustc``
+   - ``rustdoc``
```

Comment:
Rather than with notes, once you integrate with sphinx-needs you should have a parent requirement saying the `rustc`, `rustdoc`, and `cargo` (if present) binaries should be checked, and have the actual requirements as a child of that requirement.

Reply:
ccc7515e918dcd9cd88f58ce969aa0a9a125dacc

Reply:
ccc7515e918dcd9cd88f58ce969aa0a9a125dacc